tests/framework/base/SecurityTest.php

```diff
@@ -810,6 +810,28 @@ public function testGenerateRandomKey()
         $this->assertTrue($key1 != $key2);
     }
 
+    public function testGenerateRandomKeyURandom()
+    {
+        if (function_exists('random_bytes')) {
+            $this->markTestSkipped('This test can only work on platforms where random_bytes() function does not exist. You may disable it in php.ini for testing. http://php.net/manual/en/ini.core.php#ini.disable-functions');
```

Comment:
thanks for pointing out ini.disable-functions. it actually provides a way to automatically test more of the branches. i will try it out in a separate branch